bsconfig.json

```diff
@@ -2,9 +2,9 @@
   "name": "@glennsl/bs-jest",
   "namespace": true,
   "bsc-flags": ["-bs-no-version-header", "-bs-super-errors"],
-  "suffix": ".bs.js",
+  "suffix": ".mjs",
   "package-specs": {
-    "module": "commonjs",
+    "module": "es6",
```

Comment:
Is es6 the recommended module format for rescript now?

Reply:
While my personal preferences is ES6, CommonJS is still the recommended format.  I'll switch bsconfig.json back to commons but suggest keeping the babel and jest configs as these make the transition to ES6 easier.

Reply:
Defaulted to commonJS